frontend/javascripts/oxalis/view/action_bar_view.tsx

```diff
@@ -59,6 +64,75 @@ type State = {
   isNewLayoutModalOpen: boolean;
 };
 
+function AdditionalCoordinatesInputView() {
+  const additionalCoordinatesWithBounds = useSelector((state: OxalisState) =>
+    getUnifiedAdditionalCoordinates(state.dataset),
+  );
+  const additionalCoordinates = useSelector(
+    (state: OxalisState) => state.flycam.additionalCoordinates,
+  );
+  const dispatch = useDispatch();
+  const changeAdditionalCoordinates = (values: AdditionalCoordinate[] | null) => {
+    if (values != null) {
+      dispatch(setAdditionalCoordinatesAction(values));
+    }
+  };
+  const changeAdditionalCoordinatesFromVector = (values: number[]) => {
+    if (additionalCoordinates != null) {
+      dispatch(
+        setAdditionalCoordinatesAction(
+          additionalCoordinates.map((el, index) => ({
+            ...el,
+            value: values[index],
+          })),
+        ),
+      );
+    }
+  };
+
+  if (additionalCoordinates == null || additionalCoordinates.length === 0) {
+    return null;
+  }
+  return (
+    <Popover
+      content={
+        <div>
+          {additionalCoordinates.map((coord, idx) => {
+            const { bounds } = additionalCoordinatesWithBounds[coord.name];
+            return (
+              <NumberSliderSetting
+                label={coord.name}
+                key={coord.name}
+                min={bounds[0]}
+                max={bounds[1]}
+                value={coord.value}
+                spans={[2, 18, 4]}
+                onChange={(newCoord) => {
+                  const newCoords = additionalCoordinates.slice();
+                  newCoords[idx] = {
+                    ...newCoords[idx],
+                    value: newCoord,
+                  };
+                  changeAdditionalCoordinates(newCoords);
+                }}
+              />
+            );
+          })}
+        </div>
+      }
+    >
+      <ArbitraryVectorInput
+        autoSize
+        vectorLength={additionalCoordinates.length}
+        value={additionalCoordinates.map((el) => el.value)}
+        onChange={changeAdditionalCoordinatesFromVector}
+        style={{ marginLeft: 10, marginRight: 10 }}
+        addonBefore={additionalCoordinates.map((coord) => coord.name).join("")}
```

Comment:
Should this be joined with `", "`, instead maybe so the names are displayed similarly to the values?

Reply:
I would find something like `uvw` intuitive (but maybe I'm biased) and more importantly: shorter than `u, v, w`. Therefore, I'd keep it. >= 5d datasets will be very rare, anyway.

Reply:
I was thinking about the case where the dimensions are not named with single characters, for example, time and angle. Still, I agree that it's not important for now :+1: